### ISSUE TITLE
Write response comments for AU, CA, CM, CP, MA, MP, PS, RA

### ISSUE THREAD

**Issue by brittag** [18F/cg-product#432]
In order for our next JAB Q&A meeting to run smoothly and effectively, we need to review their comments ahead of time and write response comments.

Ideally we should send our PMO a copy of our response comments by end of day Friday (10/28).

We are writing comments in [this copy of the workbook](https://docs.google.com/spreadsheets/d/10Q5fCOfk0Ft41YaXLFVcGjsxKOT8_KsD8ZtfGjyZ3q0/edit).

**Acceptance criteria**

We have comments for all the following:
- [x] 34 AU items
- [x] 3 CA items
- [x] 23 CM items
- [x] 17 CP items
- [x] 2 MA items
- [x] 0 MP items
- [x] 3 PS items
- [x] 9 RA items

And:
- [x] We've emailed the updated workbook to our PMO

**Comment by brittag** [comment]
Bret and I worked on AU together and filled out almost all of it. AU has one TODO item that needs more info (AU-12).

I wrote draft responses for most of the CA, CM, CP, MA, PS, and RA comments. My responses need review for technical and operational accuracy. I also put in about a dozen "TODO" responses for comments where I had no idea what to say, mostly related to specifics of Cloud Operations work or comments that were really confusing.

In other words: this needs more work / another meeting!

**Comment by mogul** [comment]
We weren't able to meet on this Friday; will do my best to look over what
Britta put in soon, though!

**Comment by brittag** [comment]
Scheduled another meeting for this today! We should send these by COB today.

**Comment by brittag** [comment]
We finished this as much as we could! Going to send it to our PMO.

RA-3 needs consultation from Noah, so we marked that item as essentially "we need to get back to you on this one after talking to our 18F AO".
